Gopkg.toml

```diff
@@ -101,3 +101,8 @@ required = [
 [[constraint]]
   name = "k8s.io/kubernetes"
   version = "1.14.0"
+
+# Required for metric collection
+[[override]]
+  name = "github.com/coreos/prometheus-operator"
+  version = "=v0.29.0"
```

Comment:
Needs a newline

Reply:
I don't see any direct reference to `prometheus-operator` package. Do we need it?